src/lib.rs

```diff
@@ -183,6 +192,24 @@ pub trait ToPublicKey: MiniscriptKey {
     /// Converts an object to a public key
     fn to_public_key(&self) -> bitcoin::PublicKey;
 
+    /// Convert an object to x-only pubkey
+    fn to_x_only_pubkey(&self) -> bitcoin::schnorr::PublicKey {
+        let pk = self.to_public_key();
+        bitcoin::schnorr::PublicKey::from(pk.key)
+    }
+
+    /// Serialize the key as bytes based on script context. Used when encoding miniscript into bitcoin script
+    // / - Why do we need both [`MiniscriptKey::is_xonly()`] and [`ScriptContext::Tap`]?
```

Comment:
I think this comment is out of date .. I can't find any mention of `is_xonly` outside of this comment